### PR TITLE
Enable Raw datapath from kernel mode test

### PR DESCRIPTION
**Pull request by ami-GS**
## Description

Kernel mode test to call SetParam for setting execution context.

## Testing

see CI

## Documentation

N/A

### CODE REVIEW

File: src/test/bin/winkernel/control.cpp [hunk 1]
```diff
@@ -690,6 +690,19 @@ QuicTestCtlEvtIoDeviceControl(
...
+#if defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
+        if (UseDuoNic) {
+            QUIC_EXECUTION_CONFIG Config = Params->TestConfigurationParams.Config;
+            Status =
+                MsQuic->SetParam(
+                    nullptr,
+                    QUIC_PARAM_GLOBAL_EXECUTION_CONFIG,
+                    sizeof(Config),
+                    &Config);
+        }
```
Comment (nibanks) [inline src/test/bin/winkernel/control.cpp:703]: Change this to be closer to what user mode does?
```suggestion
        QUIC_EXECUTION_CONFIG Config = Params->TestConfigurationParams.Config;
        if (Config->Flags != QUIC_EXECUTION_CONFIG_FLAG_NONE) {
            Status =
                MsQuic->SetParam(
                    nullptr,
                    QUIC_PARAM_GLOBAL_EXECUTION_CONFIG,
                    sizeof(Config),
                    &Config);
        }
```

Comment (ami-GS) [reply src/test/bin/winkernel/control.cpp]: LGTM!

File: src/perf/lib/SecNetPerfMain.cpp [hunk 2]
```diff
@@ -142,10 +142,9 @@ QuicMainStart(
     QUIC_EXECUTION_CONFIG* Config = (QUIC_EXECUTION_CONFIG*)RawConfig;
     Config->PollingIdleTimeoutUs = 0; // Default to no polling.
     bool SetConfig = false;
-
-#ifndef _KERNEL_MODE
     const char* IoMode = GetValue(argc, argv, "io");
 
+#ifndef _KERNEL_MODE
```
Comment (nibanks) [inline src/perf/lib/SecNetPerfMain.cpp:147]: Actually, I _think_ the only thing that wouldn't be supported in kernel mode after all this is the `rio` stuff. Everything else can be used for kernel.

Comment (nibanks) [reply src/perf/lib/SecNetPerfMain.cpp:147]: You can exclude it for now, but depending on the final implementation, that might be common code and supported as well.

**Review by nibanks — APPROVED** [review]

**Review by nibanks — APPROVED** [review]

**Pull request merged**